contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -2648,6 +2653,102 @@ rename_update_bbf_catalog(RenameStmt *stmt)
 	}
 }
 
+static void
+rename_object_update_bbf_schema_permission_catalog(RenameStmt *stmt, int rename_type)
```

Comment:
Please add function level comment about the purpose

Reply:
Done.